corehq/apps/sms/api.py

```diff
@@ -559,7 +559,7 @@ def process_sms_registration(msg):
 
 def incoming(phone_number, text, backend_api, timestamp=None,
              domain_scope=None, backend_message_id=None,
-             raw_text=None, backend_id=None, media_url=None):
+             raw_text=None, backend_id=None, media_urls=[]):
```

Comment:
Sorry for the confusion in my original recommendation. The default value should be an immutable object so it is not accidentally mutated. (Python function parameter defaults have the same scope as the function object, so they are effectively _shared_ between invocations.) Could use an empty tuple:
```suggestion
             raw_text=None, backend_id=None, media_urls=()):
```
Or alternately, could default to `None` and change below
```diff
- msg.custom_metadata = {"media_urls": media_urls}
+ msg.custom_metadata = {"media_urls": media_urls or []}
```
Your choice.

Reply:
Are there any tests for incoming messages with multimedia? That is, tests that exercise this `media_urls`-related code?

Reply:
Ok, I've set `None ` as a default value.

Reply:
` media_urls or []` is not necessary since we already have one `IF `statement above that line:
```
If media_urls:
...
```

Reply:
> Are there any tests for incoming messages with multimedia? That is, tests that exercise this `media_urls`-related code?

I've added a new test called _testMultimediaSubmission_ in _corehq/apps/sms/tests/inbound_handlers.py_